packages/ag-charts-community/src/module/axisModules.ts

```diff
@@ -76,6 +75,14 @@ export const timeAxisOptionsDefs: OptionsDefs<AgContinuousTimeAxisOptions> = {
     crosshair: cartesianAxisCrosshairOptions(true),
 };
 
+// @todo(AG-14472) - Remove padding options
+// @ts-expect-error Remove
+timeAxisOptionsDefs.paddingInner = positiveNumber;
+// @ts-expect-error Remove
+timeAxisOptionsDefs.paddingOuter = positiveNumber;
+// @ts-expect-error Remove
+timeAxisOptionsDefs.groupPaddingInner = positiveNumber;
+
```

Comment:
Nit: Move these to the end of the file, `@ts-expect-error` actually impacts more than the ned line.